typings/index.d.ts

```diff
@@ -1142,16 +1120,14 @@ declare module 'discord.js' {
   export class InteractionWebhook extends PartialWebhookMixin() {
     constructor(client: Client, id: Snowflake, token: string);
     public token: string;
-    public send(
-      content: string | (InteractionReplyOptions & { split?: false }) | MessageAdditions,
-    ): Promise<Message | RawMessage>;
+    public send(content: string | (InteractionReplyOptions & { split?: false })): Promise<Message | RawMessage>;
     public send(options: InteractionReplyOptions & { split: true | SplitOptions }): Promise<(Message | RawMessage)[]>;
     public send(
       options: InteractionReplyOptions | APIMessage,
     ): Promise<Message | RawMessage | (Message | RawMessage)[]>;
     public send(
       content: string | null,
-      options: (InteractionReplyOptions & { split?: false }) | MessageAdditions,
+      options: InteractionReplyOptions & { split?: false },
     ): Promise<Message | RawMessage>;
     public send(
       content: string | null,
```

Comment:
This can never happen

Reply:
Should also be resolved within the latest commits